### PR TITLE
Bump Rails dependency to allow Rails 5 beta and above

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

**Comment by prathamesh-sonpatki** [comment]
@kaspth Can you help me understand the issue here? Looks like `RecordTagHelper` from this gem does not get included at all after upgrade to Rails 5 beta. I  tried renaming the module but not sure if it's the right approach.

File: record_tag_helper.gemspec [hunk 1]
```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['CHANGELOG.md', 'LICENSE', 'README.md', 'lib/**/*']
 
-  spec.add_dependency 'rails', '~> 5.0.0.alpha'
+  spec.add_dependency 'rails', '>= 5.0.0.beta1', '< 6.0'
```
Comment (kaspth) [inline record_tag_helper.gemspec:20]: We should change this to depend on Railties instead of the whole Rails package.

Comment (todd) [inline record_tag_helper.gemspec:20]: :+1: 

Additionally, we should remove the git dependencies for Rails and Arel from the Gemfile.

**Comment by kaspth** [comment]
@prathamesh-sonpatki seems like Travis is having trouble installing Rack 2.0.0.alpha, slimming the dependency to Railties should hopefully fix that 👍

**Comment by kaspth** [comment]
@todd agreed. I think with those things all in order we can even go ahead and slap some 1.0.0 :lipstick: on this :pig2:

**Comment by todd** [comment]
@kaspth I tried changing the dependency in the gemspec to railties, but that didn't solve the issue running tests for me on my machine.

File: record_tag_helper.gemspec (hunk 1 above)
Comment (todd) [inline record_tag_helper.gemspec:20]: Also, we need to change the minimum Ruby version to 2.2.2.

**Comment by kaspth** [comment]
@todd interesting. Let me try it locally.

File: record_tag_helper.gemspec (hunk 1 above)
Comment (kaspth) [inline record_tag_helper.gemspec:20]: Won't that be satisfied through Railties?

**Comment by todd** [comment]
@kaspth You can reproduce with my branch: https://github.com/todd/record_tag_helper/tree/patch-1

File: record_tag_helper.gemspec (hunk 1 above)
Comment (todd) [inline record_tag_helper.gemspec:20]: Yeah, so I guess we should just remove the minimum Ruby version constraint in this gemspec.

**Comment by todd** [comment]
For what it's worth, this looks like a different issue than the installing Rack 2.0 one.

**Comment by kaspth** [comment]
@todd this works in my Rails 5 beta app: https://github.com/rails/record_tag_helper/pull/3

**Comment by todd** [comment]
Closing in favor of #4.

**Pull request closed without merging**